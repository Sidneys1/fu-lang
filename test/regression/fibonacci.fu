fib: u32(n: u32) = {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
};

main: u32() => fib(12); // 144
