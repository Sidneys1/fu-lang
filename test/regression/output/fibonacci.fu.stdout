----------- test\regression\fibonacci.fu -----------
[2m   3 |   return fib([0;2mn - 1[2m) + fib(n - 2);[0m[0;0m <-- Checks for infix operator '-' are not implemented! [0;2m(test\regression\fibonacci.fu:3:14-18)
[0m[m[0;2m-----+[0m
[2m   3 |   return fib(n - 1) + fib([0;2mn - 2[2m);[0m[0;0m <-- Checks for infix operator '-' are not implemented! [0;2m(test\regression\fibonacci.fu:3:27-31)
[0m[m[0;2m-----+[0m
[2m   3 |   return [0;2mfib(n - 1) + fib(n - 2);[2m[0m[0;0m <-- Checks for infix operator '+' are not implemented! [0;2m(test\regression\fibonacci.fu:3:10-33)
[0m[m